packages/build/tests/tracing/tests.js

```diff
@@ -1,8 +1,14 @@
-import { trace, TraceFlags } from '@opentelemetry/api'
+import { writeFile, mkdir, rm } from 'fs/promises'
+import { fileURLToPath } from 'url'
+
+import { trace, TraceFlags, propagation } from '@opentelemetry/api'
 import { getBaggage } from '@opentelemetry/api/build/src/baggage/context-helpers.js'
 import test from 'ava'
 
-import { setMultiSpanAttributes, startTracing, stopTracing } from '../../lib/tracing/main.js'
+import { setMultiSpanAttributes, startTracing, stopTracing, loadBaggageFromFile } from '../../lib/tracing/main.js'
+
+const FIXTURES_DIR = fileURLToPath(new URL('fixtures', import.meta.url))
```

Comment:
Not a blocker, but since we're writing from this test file into this dir and then cleaning it up we might want to instead generate a temporary directory:
- https://github.com/netlify/build/blob/e75118c22685eaaf5ab471d7bd0577723cb064ac/packages/build-info/tests/helpers.ts#L20

Reply:
addressed in: https://github.com/netlify/build/pull/5343/commits/29c85648418afef5e9b22bc781c482e52e61ce03